### ISSUE TITLE
Focus not visible (accessibility issue)

### ISSUE THREAD

**Issue by DerZyklop** [the-via/app#283]
When i try to use the VIA app with my keyboard like using ⇥ to navigate through the UI I can not see where the focus is. 

I assume the bug is implemented [here](https://github.com/the-via/app/blob/2c5da8819c8889d96db659d17fb42634526d8db3/src/Routes.tsx#L16).

**Comment by ashbork** [comment]
Hi! I checked this out and the problem is even worse than it first appears - the whole app isn't usable with a keyboard only; keys on the keyboard and in the keymap pickers aren't even `buttons`.

This breaks [WCAG 2.1.1](https://www.w3.org/TR/WCAG22/#keyboard) and [WCAG 2.4.7](https://www.w3.org/TR/WCAG22/#focus-visible) by proxy. A quick fix for only the mentioned issue would be just adding a focus outline to the elements that *are* keyboard-navigable, but this requires more attention overall
